.github/workflows/main.yaml

```diff
@@ -134,6 +134,24 @@ jobs:
             exit 1
           fi
 
+          output=$(helm template oci://ttl.sh/automated-${{ github.run_id }}/replicated --version 0.0.0)
```

Comment:
this is the same as the very first helm template command. wanna move checking status informers there?